src/app/BreedProcessor.ts

```diff
@@ -103,29 +102,35 @@ export class BreedProcessor {
     return herdGrow;
   }
 
-  private returnToBank(predator: Predator, herd: Herd): void {
+  private isPredator(animal: AnimalNames): boolean {
+    return animal === AnimalNames.FOX || animal === AnimalNames.WOLF;
```

Comment:
Theoretically, it should come from AnimalRoles.PREDATOR

Reply:
changed to be included in predators[]